app/src/account.c

```diff
@@ -224,27 +223,6 @@ zxerr_t slot_parseSlot(uint8_t *buffer, uint16_t bufferLen) {
         return zxerr_out_of_bounds;
     }
 
-    //account validation
-    if (!path_is_empty(tmp_slot.path)) { //no validation required if the new slot should be empty
-        uint64_t account = 0;
-        //LE <-> BE conversion necessary to validate account
-        for(int i=0; i<SLOT_ACCOUNT_SIZE; i++) account = 256*account + tmp_slot.account.data[i];
-
-        if (path_is_mainnet(tmp_slot.path)) {
-            if (!validateChainAddress(CODEWORD_MAINNET, account)) {
-                zemu_log_stack("invalid account");
-                return zxerr_out_of_bounds;
-            }
-        }
-        if (path_is_testnet(tmp_slot.path)) {
-            if (!validateChainAddress(CODEWORD_TESTNET, account) && 
-                    !validateChainAddress(CODEWORD_EMULATORNET, account)) {
-                zemu_log_stack("invalid account");
-                return zxerr_out_of_bounds;
-            }
-        }
-    }
-
```

Comment:
I think any information that ledger gets from the outside and represents an account address should be validates using `vaildateChainAddress`. Is there a reason for deleting the validation here?